### DISCUSSION TITLE
Field Group: Reordering of fields should be stored per group, not overall

### DISCUSSION THREAD

**Discussion by nep** [ExpressionEngine/ExpressionEngine#3918] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
The current functionality of re-ordering fields can only be done within a field group.  However, the value is stored as an overall value. So if you re-order fields within one field group, the order within another field group is changed.  This is an even greater problem in the case of large and small groups.  For example:

https://example.com/system/index.php?/cp/fields&group_id=1

Group 1:
ABCDEFGHIJ

Group 2:
FGH

If you re-order Group 2 to be HGF, they get the order values 1,2,3.  At this point the order of Group 1 becomes:
AH BG CF DE IJ
(spaces for clarity)

If you then re-order Group 1 back to alphabetical, Group 2 gets re-ordered as well because the order values of F,G,H become 6,7,8. Probably not expected behavior.

**Describe the solution you'd like**
Field Order should be stored in the Field Group table, not in the Field table, and ordering should be per field group.
This is a minor but significant model and database change, but does not affect the basic functionality or the UI at all.

**Describe alternatives you've considered**
You could leave it as is. It isn't broken, just janky.

**Teachability, Documentation, Adoption, Migration Strategy**
It would be nice to add a screen shot to the docs.

**Comment by intoeetive** [comment]
Note to self: this is happening if the field belongs to several field groups at the same time
